controllers/tasks.js

```diff
@@ -43,6 +43,33 @@ const fetchTasks = async (req, res) => {
     return res.boom.badImplementation('An internal server error occurred')
   }
 }
+
+/**
+ * Fetches all the tasks of the requested user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const getUserTasks = async (req, res) => {
+  try {
+    const { username } = req.params
+    const allTasks = await tasks.fetchUserTasks(username)
+
+    if (allTasks) {
+      return res.json({
+        message: 'Tasks returned successfully!',
+        tasks: allTasks.length > 0 ? allTasks : []
+      })
+    }
+
+    return res.boom.notFound('User doesn\'t exist')
```

Comment:
Why do we say that the user does not exist, when they simply don't have any tasks assigned to them?
We can have users without any tasks, right?